main.py

```diff
@@ -1,11 +1,28 @@
-from fastapi import FastAPI
+import time
+from typing import Any
+
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+
+from searchapi import query_gemini
 
 app = FastAPI(
     title="PyCon JP 2025 Camp Tutorial API",
     description="PyCon JP 2025 Camp Tutorialの API サーバー",
     version="0.1.0",
 )
 
+# 仮の認証キー（実際の運用では環境変数などから取得すべき）
+AUTH_KEY = "pyconjp2025"
+
+
+class SingleRequest(BaseModel):
+    """単一の問い合わせリクエスト"""
+
+    key: str
+    q: str
+    options: dict[str, Any] | None = None
```

Comment:
```suggestion
    options: dict[str] | None = None
```

これで `dict[str, Any]` と等価なはず

Reply:
これは、無理では？
以下になると思う。

```suggestion
    options: dict | None = None
```